<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BASIC int 36116150D - Jonathan Iglesias Naveiras</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QPushButton" name="btnGuardar">
         <property name="text">
          <string>Guardar</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="btnCargar">
         <property name="text">
          <string>Cargar</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="btnParse">
         <property name="text">
          <string>Parsear</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QPushButton" name="btnEjecuta">
         <property name="text">
          <string>Ejecutar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="widget_3" native="true">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>410</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPlainTextEdit" name="plainTextEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="tabStopWidth">
          <number>12</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QWidget" name="widget_2" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>200</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>300</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPlainTextEdit" name="txtConsole">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>270</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="tabStopWidth">
          <number>12</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnClearConsole">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>Limpiar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>676</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Archivo">
    <property name="title">
     <string>&amp;Archivo</string>
    </property>
    <addaction name="actionNuevo"/>
    <addaction name="actionCargar"/>
    <addaction name="separator"/>
    <addaction name="actionGuardar"/>
    <addaction name="actionGuardar_como"/>
   </widget>
   <widget class="QMenu" name="menuAcerca_de">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAcerca_de"/>
   </widget>
   <addaction name="menu_Archivo"/>
   <addaction name="menuAcerca_de"/>
  </widget>
  <action name="actionPbOnClick">
   <property name="text">
    <string>pbOnClick</string>
   </property>
  </action>
  <action name="actionLimpiarOnClick">
   <property name="text">
    <string>LimpiarOnClick</string>
   </property>
  </action>
  <action name="actionParseOnClick">
   <property name="text">
    <string>ParseOnClick</string>
   </property>
   <property name="toolTip">
    <string>ParseOnClick</string>
   </property>
  </action>
  <action name="actionEjecutaOnClick">
   <property name="text">
    <string>EjecutaOnClick</string>
   </property>
  </action>
  <action name="actionGuardarOnClick">
   <property name="text">
    <string>GuardarOnClick</string>
   </property>
  </action>
  <action name="actionCargar">
   <property name="text">
    <string>Cargar</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="actionGuardar_como">
   <property name="text">
    <string>Guardar como...</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="text">
    <string>Acerca de...</string>
   </property>
  </action>
  <action name="actionNuevo">
   <property name="text">
    <string>&amp;Nuevo</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="4" margin="2"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnCargar</sender>
   <signal>clicked()</signal>
   <receiver>actionPbOnClick</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClearConsole</sender>
   <signal>clicked()</signal>
   <receiver>actionLimpiarOnClick</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>624</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnParse</sender>
   <signal>clicked()</signal>
   <receiver>actionParseOnClick</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>539</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnEjecuta</sender>
   <signal>clicked()</signal>
   <receiver>actionEjecutaOnClick</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>620</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnGuardar</sender>
   <signal>clicked()</signal>
   <receiver>actionGuardarOnClick</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
